content/en/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions.md

```diff
@@ -673,6 +673,8 @@ spec:
                 cronSpec:
                   type: string
                   pattern: '^(\d+|\*)(/\d+)?(\s+(\d+|\*)(/\d+)?){4}$'
+                 image:
```

Comment:
spacing is wrong on this line

Reply:
@mokua , Would you fix the indentation at line 676. Thanks!

Reply:
@kbhawkey done.

Reply:
Thanks @mokua . One more request: Are you able to rebase to one commit?